### ISSUE TITLE
Should returning false rather than throwing an exception be better behaviour?

### ISSUE THREAD

**Issue by Ajay100000** [EngRajabi/Enum.Source.Generator#14]
The code looks very good, but I think it is more useful if 'IsDefinedFast(string states)' returns false rather than throwing an exception. This means that if the string is not defined in the enum collection and we get back 'false' and can act on that in the if statement.

Here is my example:

```
// currently throws an exception (would be better if it returned false)
if (UserTypeTestEnumExtensions.IsDefinedFast("Gien"))
{
    System.Console.WriteLine("'Gien' is defined");
}
else
{
    System.Console.WriteLine("Not defined is 'Gien'"); 
}


Here is the pseudo code for what I am suggesting:

public static bool IsDefinedFast(string states)
{
    return states switch
    {
        nameof(UnitTests.UserTypeTest.Men) => true,
        nameof(UnitTests.UserTypeTest.Women) => true,
        nameof(UnitTests.UserTypeTest.None) => true,
        _ => false
    };
}

```
This is only for this specific scenario that I suggest returning false.

What do you think?

**Comment by EngRajabi** [comment]
Hello, thank you for your participation.
I am very busy at the moment and cannot apply these changes. I will be happy if you can do it.

**Comment by EngRajabi** [comment]
done

**Comment by Ajay100000** [comment]
Thanks, that is really good. Will this be released as a package? v1.0.12 seems to be from may 31st.

Thanks again, this will be very useful.

**Comment by EngRajabi** [comment]
> Thanks, that is really good. Will this be released as a package? v1.0.12 seems to be from may 31st.
> 
> Thanks again, this will be very useful.

Yes, the version is finalized in 1.0.13. There should be no problem

**Comment by Ajay100000** [comment]
I have tried 1.0.13 out. It works perfectly. Thanks again for making this change.
